db/migrate/20180819185350_add_fields_to_relief_camp.rb

```diff
@@ -0,0 +1,12 @@
+class AddFieldsToReliefCamp < ActiveRecord::Migration[5.2]
+  def change
+    add_column :relief_camps, :latitude, :float
+    add_column :relief_camps, :longitude, :float
+    add_column :relief_camps, :address, :text
+    add_column :relief_camps, :phone_no, :string
+    add_column :relief_camps, :whatsapp_no, :string
+    add_column :relief_camps, :email, :string
+    add_column :relief_camps, :contact_person, :string
+    add_column :relief_camps, :coordinator_id, :sting
```

Comment:
typo here.. should be string

Reply:
it already has a `coordinator_id`

Reply:
@sparkymat the type should be string..it says sting
```
:st  r  ing 
     ^
     |
```